tests/wallet/test_nft_store.py

```diff
@@ -33,5 +35,18 @@ async def test_nft_store(self) -> None:
             assert nft == (await db.get_nft_list(wallet_id=uint32(1), did_id=a_bytes32))[0]
             assert nft == await db.get_nft_by_id(a_bytes32)
             # Test delete
-            await db.delete_nft(a_bytes32)
+            await db.delete_nft(a_bytes32, uint32(11))
             assert await db.get_nft_by_id(a_bytes32) is None
+            # Test reorg
+            nft1 = NFTCoinInfo(
+                a_bytes32_1,
+                Coin(a_bytes32, a_bytes32, uint64(1)),
+                LineageProof(a_bytes32, a_bytes32, uint64(1)),
+                puzzle,
+                uint32(1),
+                uint32(12),
+            )
+            await db.save_nft(uint32(1), a_bytes32_1, nft1)
+            assert nft1 == (await db.get_nft_list(wallet_id=uint32(1)))[0]
+            await db.rollback_to_block(10)
```

Comment:
I would think we want to have more than a single test-case for this function. Ideally the tests would be split up into separate functions too.